test/ci-cd/terraform/apply.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/env bash
+
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+MY_PATH="$(
+    cd "$(dirname "$0")" >/dev/null 2>&1
+    pwd -P
+)"
+
+terraservices=("cloudbuild")
+
+cd "${MY_PATH}/initialize" &&
+    terraform init &&
+    terraform plan -input=false -out=tfplan &&
+    terraform apply -input=false tfplan || exit 1
+rm tfplan
```

Comment:
Worth adding this to the `terraservice` list above in order to reduce duplication?

Reply:
Typically I make the `initialize` idempotent to be able to run it via Cloud Build. I ended up removing that logic since this configuration persists. I can probably just add it to the array now.